accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -274,6 +274,7 @@ leaves, and nothing else.
 ```dart
 extension type TinyJson(Object it) {
   Iterable<num> get leaves sync* {
+    final it = this.it; // To get promotion.
```

Comment:
hope it will be `val` one day.

Reply:
Indeed, cf. https://github.com/dart-lang/language/issues/3332. ;-)